ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml

```diff
@@ -160,10 +160,6 @@ objects:
         mkdir -p /go/src/github.com/openshift/
         cp -r /tmp/shared/openshift-azure /go/src/github.com/openshift/
         cd /go/src/github.com/openshift/openshift-azure/
-        mkdir -p /go/src/github.com/openshift/openshift-azure/secrets
```

Comment:
Why this one is removed? we still need these in the plugin config

Reply:
They are already set in the setup container.

Reply:
But we need them for Upgrade tests. because until we refactor pluginConfig Upgrade is initiating NewPlugin which reads those files.

Reply:
At the current state Upgrade needs almost all the save pre-reqs as Create. otherwise E2E upgrade test fails

Reply:
The files are already copied in the secrets directory during setup, there is no need to copy them again here.